tests/PayloadListener.Tests/Services/EventPayloadRecieverServiceTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using NUnit.Framework;
```

Comment:
missing copyright header

Reply:
May also want to sort the usings; There's a Visual Studio extension `CodeMaid` for that.